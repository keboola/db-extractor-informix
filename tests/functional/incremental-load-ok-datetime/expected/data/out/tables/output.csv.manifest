{
    "destination": "output",
    "incremental": true,
    "metadata": [
        {
            "key": "KBC.name",
            "value": "incremental"
        },
        {
            "key": "KBC.sanitizedName",
            "value": "incremental"
        },
        {
            "key": "KBC.schema",
            "value": "%A"
        },
        {
            "key": "KBC.type",
            "value": "table"
        }
    ],
    "column_metadata": {
        "id": %A,
        "int": %A,
        "decimal": %A,
        "text": %A,
        "date": %A,
        "datetime": %A
    },
    "columns": [
        "id",
        "int",
        "decimal",
        "text",
        "date",
        "datetime"
    ]
}
