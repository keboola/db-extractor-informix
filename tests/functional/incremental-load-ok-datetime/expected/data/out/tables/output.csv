"1","80","10.1","bBC","2020-10-03","2020-10-01 10:00:00.000"
"3","60","20.3","aBC","2020-10-01","2020-10-02 10:00:00.000"
"2","70","50.2","3BC","2020-10-02","2020-10-03 10:00:00.000"
"5","40","60.5","cBC","2020-10-06","2020-10-04 10:00:00.000"
"8","10","70.8","eBC","2020-10-07","2020-10-05 10:00:00.000"
"7","20","40.7","dBC","2020-10-08","2020-10-06 10:00:00.000"
"6","30","30.6","2BC","2020-10-04","2020-10-07 10:00:00.000"
"4","50","80.4","1BC","2020-10-05","2020-10-08 10:00:00.000"
